upstream-sha256: 1aff9db9b1e0882838f4c43c69e530df0192c8ac180fd41a132e7999ba85c06f
actual-sha256: 416dadb9c8f1c3f47415acf11d7302cd052ffc24106265efe911c17a0ed5f68b
--- upstream.expected
+++ tolk.actual
@@ -16,5 +16,5 @@
   float val9 = (*(data1_1000+(alu0+900)));
   int alu1 = (lidx1+(gidx0<<1)+(gidx1*20)+(lidx0*10));
   float val10 = (*(data2_100+alu1));
-  *(data0_100+alu1) = (val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
+  *(data0_100+alu1) = (0.0f+val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
 }
